backend/pkg/helm/handler.go

```diff
@@ -39,7 +39,10 @@ func NewActionConfig(clientConfig clientcmd.ClientConfig, namespace string) (*ac
 		namespace:    namespace,
 	}
 	logger := func(format string, a ...interface{}) {
-		log.Info().Str("namespace", namespace).Msg(format + "\n" + fmt.Sprintf("%v", a))
+		zlog.Info().
```

Comment:
I think instead of printing the action manually you can create a helper function which internally uses the golang runtime packages [Caller](https://pkg.go.dev/runtime#Caller) function to print the source of the log. That can be helpful in identifying the source of the log very easily.

Reply:
Let me look into it